src/components/components/CandidateProfileCard/CandidateProfileCard.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import "./CandidateProfileCard.scss";
+import CandidateImage from "./assets/candidateImage.svg";
+
+function CandidateProfileCard(props) {
+  const { candidate } = props;
```

Comment:
we can get this directly by putting
function CandidateProfileCard( { candidate } ) {
in here